components/report/reportContents.tsx

```diff
@@ -8,16 +8,40 @@ import ChartCard from 'components/report/chart-card';
 import { BankAccount } from 'integrations/stitch/types';
 import TopExpensesCard from './top-expenses-card';
 import TopDebitOrderCard from 'components/report/top-debit-order-card';
+import AccountsCard from './bank-accounts';
 
-export function ReportContents(props: { bankAccount: BankAccount }): JSX.Element {
+export function ReportContents(props: { bankAccounts: BankAccount[]}): JSX.Element {
+    const bankAccounts = props.bankAccounts.filter(x => x.accountType === 'cheque' || x.accountType === 'current') ?? props.bankAccounts.find(x => x.accountType === 'savings');
+    const bankAccount = bankAccounts.sort(b => b.currentBalance.quantity).reverse()[0];
```

Comment:
Do you think it reads better if we use a binary function in `sort` instead of `reverse`? I'm not convinced it would be better, binary sorting functions always force me to stop and think a bit.

```suggestion
    const bankAccount = bankAccounts.sort(a, b => b.currentBalance.quantity - a.currentBalance.quantity)[0];
```